indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -51,16 +52,30 @@ class RHManageRegistrationFormsDisplay(RHManageRegFormsBase):
 
     def _process(self):
         regforms = sorted(self.event.registration_forms, key=lambda f: f.title.lower())
-        form = ParticipantsDisplayForm()
+        forbidden_visibility_forms = []
```

Comment:
a set would be the better choice here

Reply:
This breaks the current way that the list is passed to the JS (`{{ forbidden_visibility_forms|tojson }}`). What Is the best way to solve this? I guess we could either iterate the set server-side in jinja, or convert to list before passing it to the template

Reply:
You can use `...|list|tojson` in the template